helpers/parse_args.go

```diff
@@ -82,6 +86,10 @@ func ParseArgs() error {
 	if os.Getenv("AZ_SAMPLES_KEEP_RESOURCES") == "1" {
 		keepResources = true
 	}
+	armEndpointString = os.Getenv("AZ_ARM_ENDPOINT")
+	tenantID = os.Getenv("AZ_TENANT_ID")
```

Comment:
This stuff is already implemeted in the `iam` package

Reply:
removed the duplicate parameters